jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/bsm/BsmCoder.java

```diff
@@ -50,6 +52,7 @@ public void decodeFromHexAndPublish(InputStream is, String topic)
 
       try (Scanner scanner = new Scanner(is)) {
          
+    	 data.info("Attempting to decode message");

```

Comment:
No data logging necessary. Make log messages concise. If you have logged a file in the beginning of a stage, it does not need to be logged again within that stage. Remember this is not a log file for troubleshooting code. It is for tracking files. If you do log, it should be a debug level log. Also log message should be concise. Make it "Decoding messages ...". A file contains one or more messages.